src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -425,6 +440,38 @@ public void ValidateAndSetIgnoreUnexpectedElementsAndAttributesFromString(string
     this.IgnoreUnexpectedElementsAndAttributes = boolValue;
 }
 
+/// <summary>
+/// Set the CustomHttpHeaders property from the custom http headers string.
+/// </summary>
+/// <param name="stringValue">The value to set.</param>
+public void ValidateAndSetCustomHttpHeadersFromString(string headersValue) 

```

Comment:
It might make sense to split this into 2 separator methods one to validate and another to set

Reply:
Done